### ISSUE TITLE
Python taking 100% of my processor

### ISSUE THREAD

**Issue by arturbenchimol** [azai91/alfred-drive-workflow#23]
Hi there, I've been using this workflow and I absolutely love it. I realized it was taking almost 100% of my processor capacity and I had to turn it off to bring it back to normal. The process that was taking all the CPU power was Python.

It also elevated Alfred's energy impact to 151.95

I am not really a programer so I wouldn't be able to point out exactly what was happening.  Is there any fix I should do? 

thanks!

**Comment by justinjm** [comment]
Hi,

First and foremost, I am huge fan of this workflow and use it daily! However, I am experiencing the same issue as @arturbenchimol and I've turned it off also :-(

I'm also not an expert programmer but there is a process called "Python" that is consuming ~98% of my CPU. Here are the "Open files and ports" (found via Activity Monitor -> Inspect): 

```
cwd
/Users/justin/Library/Mobile Documents/com~apple~CloudDocs/Alfred 3/Alfred.alfredpreferences/workflows/user.workflow.D41D8E7C-29C4-4B87-A53B-11A0C028D529
txt
/System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
txt
/System/Library/Frameworks/Python.framework/Versions/2.7/Python
txt
/usr/lib/dyld
txt
/private/var/db/dyld/dyld_shared_cache_x86_64h
0
/dev/null
1
/dev/null
2
/dev/null
3
/Users/justin/Library/Caches/com.runningwithcrayons.Alfred-3/Workflow Data/com.drive.azai91/com.drive.azai91.log
4
/Users/justin/Library/Caches/com.runningwithcrayons.Alfred-3/Workflow Data/com.drive.azai91/com.drive.azai91.log
```

Finally, my system specs are below:

- Alfred-Drive-Workflow 1.2
- Alfred 3.3.2 [818]
- macOS Sierra 10.12.4

Please let me know what else I can do to help debug from here.

Justin

**Comment by justinjm** [comment]
@arturbenchimol FYI - give this a try:

https://www.alfredforum.com/topic/10193-python-fix-how-to-fix-python-workflows-hanging-alfred-on-sierra/

https://github.com/deanishe/alfred-fixum

**Comment by arturbenchimol** [comment]
thanks, @justinjm I'll give it a try and report back here. Thanks!

**Comment by ymar** [comment]
I have the same issue. Alfred Fixum didn't solve the issue unfortunately. Only resolution would be if the workflow get's updated. @azai91 any plans for solving this bug?

**Comment by cyaconi** [comment]
I'm facing the same problem. Hope this could be fixed soon.
Thanks!

**Comment by arturbenchimol** [comment]
I added the Fixum workflow and it stopped bothering me.

**Comment by jgarber** [comment]
Fixed in #22. Download the latest workflow to solve the problem. You'll have to manually install it from source since @azai91 hasn't released lately.

**Comment by LewisLebentz** [comment]
@jgarber is just swapping the file for the new version enough?

**Comment by cobbman** [comment]
Sorry I have to ask, but do I run "make" after cloning this to get the workflow updated? I tried it but getting some errors. I just want to get this updated because I'm having the same issue as mentioned above. Thanks!

edit: screenshot http://d3fy.xyz/GCS9r5

**Comment by cobbman** [comment]
Sorry for the noob question here, but I can't find anything online to help me with this...

How do I get this repo "packaged" into an alfred workflow so I can install it? I'm trying to get the latest version. Thanks

**Comment by cobbman** [comment]
@azai91 - please create a release with the most recent version. Thank you.

**Comment by sorbits** [comment]
It is the `src` directory which is the actual workflow, so if you clone this repository, remove the old workflow (from Alfred’s `workflows` folder) and create a symbolic link to this repository’s `src` folder.

Though currently it is still using a version of the alfred-workflow module which is incompatible with Alfred 3.4 but I have submitted a PR for updating it.

I added the new version as a git submodule so use `git clone --recursive «repository»` when you clone it.
